frontend/next-sitemap.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  siteUrl: "https://cellxgene.cziscience.com/",
+  generateRobotsTxt: true,
+  exclude: ["/collections-sitemap.xml"], // add additional directories or pages here if they should be excluded from the sitemap
+  robotsTxtOptions: {
+    policies: [
+      // {userAgent: "*", disallow: "/folderToExclude/fileToExclude"} <-- use this option if you wish to exclude certain pages or folders from robots.txt
```

Comment:
This is awesome! We can add no follow to our private collection pages here too later💡